rmgpy/statmech/torsion.pyx

```diff
@@ -540,7 +540,7 @@ cdef class HinderedRotor(Torsion):
                 A[:-1, m] = np.cos(m * angle)
                 A[:-1, numterms + m - 1] = np.sin(m * angle)
             # This row forces dV/dangle = 0 at angle = 0
-            A[N, numterms:] = 1
+            A[N, numterms:] = np.arange(1., numterms)
```

Comment:
I saw you related to this in the PR message, but could you please explain what this technically does?